R/epidatacall.R

```diff
@@ -103,42 +103,38 @@ print.epidata_call <- function(x, ...) {
   ))
 }
 
-#' Customize fetch settings
+#' Set custom API request parameters
 #'
-#' @rdname fetch_args_list
-#' @aliases fetch_args
-#'
-#' @description
-#' A constructor for `fetch_args` objects, which are used to pass arguments to
-#' the `fetch` function.
+#' Used to specify custom options when making API requests, such as to set
+#' timeouts or change data formats. These options are used by `fetch()` when it
+#' makes calls to the Epidata API.
 #'
 #' @param ... not used for values, forces later arguments to bind by name
-#' @param fields a list of epidata fields to return, or NULL to return all
-#'  fields (default) e.g. c("time_value", "value") to return only the
-#'  time_value and value fields or c("-direction") to return everything except
-#'  the direction field
-#' @param disable_date_parsing disable automatic date parsing; by default `FALSE`
-#' @param disable_data_frame_parsing disable automatic conversion to data frame; this
-#'  is only supported by endpoints that only support the 'classic' format (non-tabular).
-#'  by default `FALSE`
-#' @param return_empty boolean that allows returning an empty tibble if there is no data;
-#'  by default `FALSE`
-#' @param timeout_seconds the maximum amount of time to wait for a response; by default
-#'  `FALSE`
-#' @param base_url base URL to use; by default `NULL`, which means the global base url
-#'  `"https://api.delphi.cmu.edu/epidata/"`
-#' @param dry_run boolean that allows skipping the call to the API and instead
-#'  returns the `epidata_call` object (useful for debugging); by default `TRUE`
-#' @param debug boolean that allows returning the raw response from the API; by default
-#'  `FALSE`
-#' @param format_type the format to request from the API, one of classic, json, csv; this
-#'  is only used by `fetch_debug`, and by default is `"json"`
-#' @return
-#' - For `fetch_args_list`: a `fetch_args` object
+#' @param fields a list of epidata fields to return, or `NULL` to return all
+#'   fields (default). e.g. `c("time_value", "value")` to return only the
+#'   `time_value` and `value` fields or `c("-direction")` to return everything
+#'   except the direction field
+#' @param disable_date_parsing disable automatic date parsing; by default
+#'   `FALSE`
+#' @param disable_data_frame_parsing disable automatic conversion to data frame;
```

Comment:
Just to confirm, linebreaks in roxygen only show up in the code itself, and not when the documentation is displayed, right? either way, not sure how this was at 87; maybe we were wrapping at 100?

Reply:
Right, this all gets automatically reformatted when displayed. roxygen makes this a nice HTML list, while R I think automatically re-wraps lines based on your terminal width